.earthly_version_flag_overrides

```diff
@@ -1 +1 @@
-referenced-save-only,use-copy-include-patterns,earthly-version-arg,use-cache-command,use-host-command,check-duplicate-images,use-copy-link,parallel-load,shell-out-anywhere,new-platform,no-tar-build-output
+referenced-save-only,use-copy-include-patterns,earthly-version-arg,use-cache-command,use-host-command,check-duplicate-images,use-copy-link,parallel-load,shell-out-anywhere,new-platform,no-tar-build-output,wait-block
```

Comment:
let the dogfooding begin :sweat_smile:

Reply:
(I'm also fine if we want to keep this PR open until I'm back after vacation time)